Makefile

```diff
@@ -92,6 +93,13 @@ python_build: pip
 clean:
 	$(PYTHON) setup.py clean --all
 
+install-user:
+	$(PYTHON) setup.py install $(PYTHON_DEVELOP_ARGS)
```

Comment:
I'm  trying to understand why are you using DEVELOP_ARGS on a normal user install. IMO,  DEVELOP_ARGS should be used  when creating links. DEVELOP_ARGS is checking if the `develop` subcommand has `--user`  or not. And here you are using this with another subcommand.

Maybe I'm missing something.

Reply:
> I'm trying to understand why are you using DEVELOP_ARGS on a normal user install. IMO, DEVELOP_ARGS should be used when creating links. DEVELOP_ARGS is checking if the `develop` subcommand has `--user` or not. And here you are using this with another subcommand.
> 
> Maybe I'm missing something.

No, this should use `--user`` directly.